frontend/src/features/seeds/routes/ViewSeeds.tsx

```diff
@@ -50,23 +54,25 @@ export const ViewSeeds = () => {
   };
 
   return (
-    <PageLayout styleNames="flex flex-col space-y-4">
```

Comment:
Also here, is this on purpose? Please give separate commits for such seemingly unrelated changes in future.

Reply:
I wrapped it into the `<Suspense>` Component. This is just formatting.